### PR TITLE
Kosta/etutoring

### PR DESCRIPTION
**Pull request by kostamas**

### CODE REVIEW

File: src/locale/locale-en.json [hunk 1]
```diff
@@ -105,5 +105,56 @@
         "CATEGORY_ACCURACY": "{{categoryProgress}}% accuracy",
         "AVERAGE_TIME_CATEGORY": "Avg.: {{avgTime}} sec",
         "SHOW_DETAILS": "Show details"
+    },
+    "E_TUTORING":{
+        "HEADER_TITLE_PREFIX": "Lesson Name",
+        "COMPLETE_DIAGNOSTIC_FIRST": "Please complete the diagnostic test in order to access Zinkerz Online Educators Lessons",
+        "NO_TEACHER_TITLE" : "Want to get the best SAT prep possible?",
+        "NO_TEACHER_SUBTITLE_1": "Schedule a FREE online lesson with one of our professional tutors.",
+        "NO_TEACHER_SUBTITLE_2": "Our tutors are SAT experts who specialize in maximizing teaching opportunities across our test prep platforms.",
```
Comment (assafshp) [inline src/locale/locale-en.json]: SAT

File: src/locale/locale-en.json [hunk 2]
```diff
@@ -105,5 +105,56 @@
         "CATEGORY_ACCURACY": "{{categoryProgress}}% accuracy",
         "AVERAGE_TIME_CATEGORY": "Avg.: {{avgTime}} sec",
         "SHOW_DETAILS": "Show details"
+    },
+    "E_TUTORING":{
+        "HEADER_TITLE_PREFIX": "Lesson Name",
+        "COMPLETE_DIAGNOSTIC_FIRST": "Please complete the diagnostic test in order to access Zinkerz Online Educators Lessons",
+        "NO_TEACHER_TITLE" : "Want to get the best SAT prep possible?",
```
Comment (assafshp) [inline src/locale/locale-en.json]: SAT

File: src/components/znkChat/templates/chatter.template.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div class="chatter-wrapper"
      ng-class="{'offline': chatterObj.presence === d.userStatus.OFFLINE,
      'online': chatterObj.presence === d.userStatus.ONLINE,
-     'idle': chatterObj.presence === vm.userStatus.IDLE}">
+     'idle': chatterObj.presence === d.userStatus.IDLE}">
```
Comment (assafshp) [inline src/components/znkChat/templates/chatter.template.html]: was it a bug ?

File: src/components/assignModule/homework.service.js [hunk 4]
```diff
@@ -157,7 +161,8 @@
                         var sectionsResults = [];
                         var promArr = [];
                         var dontInit = true;
-                        if (exam.typeId !== ExamTypeEnum.MINI_TEST.enum) {
+
+                        if (exam.typeId !== ExamTypeEnum.MINI_TEST.enum || (angular.isDefined(diagnosticExamId) && exam.id === diagnosticExamId)) {
```
Comment (assafshp) [inline src/components/assignModule/homework.service.js]: are you sure?

File: Gruntfile.js [hunk 5]
```diff
@@ -167,7 +167,7 @@ module.exports = function (grunt) {
             ci: {
                 configFile: 'test/karma-unit.conf.js',
                 singleRun: true,
-                browsers: ['PhantomJS']
+                browsers: ['Chrome']
```
Comment (assafshp) [inline Gruntfile.js]: is it working ?

File: src/components/znkChat/templates/chatter.template.html (hunk 3 above)
Comment (kostamas) [inline src/components/znkChat/templates/chatter.template.html]: yes

File: Gruntfile.js (hunk 5 above)
Comment (kostamas) [inline Gruntfile.js]: yes

**Pull request merged**